src/context/lab-test-results-context.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from 'react'
+import {LabTest} from '../types/selectTest'
+
+interface LabTestResultsContextProps {
+  labTestResults: any
```

Comment:
Can we avoid using any type. Will we be able to create a type and leverage the type

Reply:
changes have been made, introduced new type